src/portable/control.rs

```diff
@@ -61,13 +61,15 @@ pub fn do_start(inst: &InstanceInfo) -> anyhow::Result<()> {
     }
 }
 
-pub fn get_server_cmd(inst: &InstanceInfo) -> anyhow::Result<process::Native> {
+pub fn get_server_cmd(inst: &InstanceInfo, is_shutdown_supported: bool)
```

Comment:
```suggestion
pub fn get_server_cmd(inst: &InstanceInfo, auto_shutdown: bool)
```

Reply:
All supported server versions support auto-shutdown.  What's the concern here?

Reply:
I think we support socket activation only for 2.0. So even if 1.x supports shutdown it's bad idea to enable shutdown there.